### PR TITLE
[timedclient] implement MaximalTimeoutSnoozeCounter

### PR DESCRIPTION
**Pull request by teleshoes**

### CODE REVIEW

**Comment by teleshoes** [comment]
previously, `timedclient-qt5 -e 'MaximalTimeoutSnoozeCounter=2'` would result in
`Maemo::Timed::Event::setAttribute`
instead of `Maemo::Timed::Event::setMaximalTimeoutSnoozeCounter`

and thus, there was no way to actually set this value with timedclient

File: tools/timedclient/timedclient.cpp [hunk 1]
```diff
@@ -1482,6 +1482,10 @@ static void do_add_event(char *args)
     {
       cur_eve.setTimezone(val);
     }
+    else if( !strcmp(key, "MaximalTimeoutSnoozeCounter") )
```
Comment (pvuorela) [inline tools/timedclient/timedclient.cpp:1485]: Maybe this should start with lower case like other event args?

**Review by pvuorela — COMMENTED** [review]
Should also list this option with --help?

cc @spiiroin

File: tools/timedclient/timedclient.cpp (hunk 1 above)
Comment (teleshoes) [reply tools/timedclient/timedclient.cpp]: +1 (i was copying the info msg, ill change both)

File: tools/timedclient/timedclient.cpp [hunk 2]
```diff
@@ -881,7 +881,7 @@ static void event_emit_details(CONST Maemo::Timed::Event &eve)
 {
   event_emit_info(eve);
 
-  printf("  MaximalTimeoutSnoozeCounter: %d\n",
+  printf("  maximalTimeoutSnoozeCounter: %d\n",
```
Comment (pvuorela) [inline tools/timedclient/timedclient.cpp:884]: I suppose this should stay upper case to be consistent with the below items. Being more kind of "pretty printed" details.

Comment (teleshoes) [reply tools/timedclient/timedclient.cpp]: ok

**Review by spiiroin — APPROVED** [review]

**Comment by spiiroin** [comment]
Included in timed >= 3.6.15

**Pull request merged**